city_scrapers/spiders/chi_ssa_2.py

```diff
@@ -0,0 +1,61 @@
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+import re
+from datetime import datetime, time
+
+class ChiSsa2Spider(CityScrapersSpider):
+    name = "chi_ssa_2"
+    agency = "Chicago Special Service Are #2 Belmont/Central"
+    timezone = "America/Chicago"
+    allowed_domains = ["belmontcentral.org"]
+    start_urls = ["http://belmontcentral.org/about-ssa-2/ssa2-meeting-minutes-audit/"]
+    location = {
+        "address": "5534 W. Belmont Avenue Chicago, IL 60641",
+        "name": "Belmont-Central Chamber of Commerce"
+    }
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        for item in response.css(".entry-content a"):
+            start = self._parse_start(item)
+            if not start:
+                continue
+            meeting = Meeting(
+                title="SSA #2 Commissioners’ Meeting",
```

Comment:
Small thing, but we've usually been returning `Commission` for the title for SSAs since it repeats less info